index.html

```diff
@@ -0,0 +1,43 @@
+           </div>
+           
+        <div class="Thewoo"> <img src="https://upload.wikimedia.org/wikipedia/commons/e/e4/Seal_of_Worcester%2C_Massachusetts.svg">
```

Comment:
Why is the class name "Thewoo"? 
Also, in the future, please use camelCase. For example, instead of "Myname", you would use "myName"